i18n/en.json

```diff
@@ -4682,6 +4682,14 @@
     "id": "model.user_access_token.is_valid.user_id.app_error",
     "translation": "Invalid user id"
   },
+  {
+    "id": "model.user.is_valid.email.app_error",
+    "translation": "Invalid email"
```

Comment:
Why is it invalid? Can we add some more details so an end user has clear next steps to correct the username or email errors?

Reply:
like what @esethna this message is similar to another error message when the email is invalid.

Reply:
@esethna updated the text, please take a look and let me know what is your thoughts